bazel/repository_locations.bzl

```diff
@@ -555,6 +556,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         # replacement for protobuf JSON.
         use_category = ["controlplane", "dataplane_core"],
         release_date = "2020-08-06",
+        # cpe:2.3:a:json-for-modern-cpp_project:json-for-modern-cpp: ?
```

Comment:
I think this is right
https://nvd.nist.gov/products/cpe/detail/394565?namingFormat=2.3&orderBy=CPEURI&keyword=json-for-modern-cpp_project&status=FINAL